### ISSUE TITLE
alex-3.1.5 update breaks gtk2hs-buildtools

### ISSUE THREAD

**Issue by juhp** [gtk2hs/gtk2hs#146]
Originally filed as fpco/stackage#1004

```
c2hs/c/CLexer.x:400:49:
    Not in scope: ‘ord’
    Perhaps you meant one of these:
      ‘or’ (imported from Prelude), ‘odd’ (imported from Prelude),
      ‘ord#’ (imported from GHC.Exts)
```

A similar issue for morte was fixed like this: Gabriel439/Haskell-Morte-Library/pull/34

**Comment by juhp** [comment]
Having said that I can't reproduce yet with alex-3.1.6 in Stackage Nightly 2015-11-30.

**Comment by hamishmack** [comment]
This was already fixed in 68e94f365366d3bc215bfbad502b66b87172cca8 and uploaded to hackage as [0.13.0.5](http://hackage.haskell.org/package/gtk2hs-buildtools).

**Comment by juhp** [comment]
Thanks Hamish - awesome - sorry for the noise. :)
